omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -57,10 +66,13 @@
     <% end -%>
 
     <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
-    location ~ /organizations/([^/]+)/data-collector {
+    location ~ "^/organizations/([^/]+)/data-collector$" {
```

Comment:
Awesome. I was totally incorrect here:

During a `chef-client` run, we do a POST to the `data-collector` endpoint at the start of the run. If that returns 404, then we disable the reporter ([link](https://github.com/chef/chef/blob/master/lib/chef/data_collector.rb#L125-L133))

Also, when using the chef-server as a data-collector endpoint, we leave off the v0 portion of the URL ([link](https://github.com/chef/chef/blob/b2b87c35ceb2d6e05c2be2f381b3e3dbe86a3dd1/chef-config/lib/chef-config/config.rb#L874-L880))